examples/pulse_and_brightness.py

```diff
@@ -2,39 +2,58 @@
 
 import time
 import sys
-from pypowermate import Powermate
+from pypowermate.powermate import Powermate
+import glob
 
 if __name__ == '__main__':
-	if len(sys.argv) != 2:
-		sys.stderr.write('usage: %s <input device>\n' % sys.argv[0])
+	if len(sys.argv) > 1 :
+		sys.stderr.write('usage: %s \n  the command has no argument' % sys.argv[0])
+		sys.exit(1)
+	powermates=glob.glob('/dev/input/by-id/*PowerMate*')
+	if len(powermates) == 0 :
+		print('no powermate found !')
```

Comment:
I'd prefer if you can still pass paths on the cmdline. Perhaps the glob could be used only if no parameters are passed?

Reply:
This was added to the master branch, thanks for the contribution.